### ISSUE TITLE
Left navigation bar should be minimizeable to the left (and back)

### ISSUE THREAD

**Issue by assafharel** [PeerSay/Atlas#127]

**Comment by Pavek** [comment]
Please present a design or muckup of minimized view.

**Comment by assafharel** [comment]
Closing it for now (we didn't find good examples to copy from...)
Anyways, it should be solved when you finish the rest of the table issues (like freezing the title, the total and the requirements, etc.)

**Comment by assafharel** [comment]
Re-open as I found a good example - Google Analytics.

**Comment by assafharel** [comment]
Re-open, as we didn't intend to tie the right side expanding-bar to the left side menu collapse button. For instance, now it's impossible to have both the menu and the right side expanded, which is my default mode...

**Comment by Pavek** [comment]
I tried having right & left 'side-bars' collapsed independently - it leads to some quite weird & unpleasantly looking layouts (e.g. menu collapsed / right-side expanded: main contnet is aligned to left). Moreover, note that majority of screens _have no_ right side-bar at all like this: https://gyazo.com/0f8331469594759049b8f45b97bd0daa

So, I decided to have 2 states in total: full-sceen & compant, each looking nice on all screens.

In other words, I can do it, but you may not like it :)

**Comment by assafharel** [comment]
Since it's only medium priority I suggest cherry picking it out of the
branch, and re-visit it again in sprint#3.

Assaf
בתאריך Thu, Aug 13, 2015 ב-18:19 מאת Pavel Zhytko <notifications@github.com

**Comment by Pavek** [comment]
By 'cherry-picking' do you mean removing this commit? Note: it contains other layout fixes too (actually, new layout is implemented), for example menu on mobile is fixed (compare to current state in production).

**Comment by assafharel** [comment]
Uhmmm. Dilemma.
Yes, I saw the fixes for mobile. Nice.
OK, so let me explain the problem again, with the current solution and let
see if there's a way out of it.
With the current solution, if the user chooses to expand the view, she
looses the left side navigation. Getting it back requires her to find that
tiny button on the left, or the expansion bar to the right. Both are not
immediate actions. Moreover, the expansion state is sticky, I.e. Going in
and out of the project or into other projects doesn't "solve" this and you
find yourself "stuck" with the expanded view without being able to get out
of it.
Now, assuming you found your way out of it, finally you have a menu bar to
the left, now you feel that 1/3 of the screen is not used, and you do want
it used (especially in the table, if you have more then 3 products), so you
find yourself expanding and collapsing all the time. Expanding to get a
good view of the table and collapsing to bring back the menu. Not a good UX
at all.
בתאריך Thu, Aug 13, 2015 ב-21:14 מאת Pavel Zhytko <notifications@github.com

**Comment by Pavek** [comment]
Implemented independent togglers -- let's see how it goes.
